src/main/java/refactoringml/RunSingleProject.java

```diff
@@ -2,6 +2,8 @@
 
```

Comment:
I think we can remove this entire class, it is redundant with the new CSV importer. The user can specify the project within a csv file and start the data-collection. This, way we don't have to maintain and describe this class anymore.

Reply:
Sounds a-ok to me